client/src/components/InputFormField/InputFormField.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Field} from 'formik';
+import { TextField } from '@mui/material';
+
+interface Props {
+  name: string;
+  type: string;
+  label: string;
+}
+
+const InputFormField: React.FC<Props> = (props) => {
```

Comment:
I think useField hook would be great here.
We can add error message right after Field component.
Reference: [Formik dock](https://formik.org/docs/tutorial#leveraging-react-context) (at the end).